tests/pytest/enrollment_switchio/test_views.py

```diff
@@ -92,7 +92,7 @@ def test_get_with_session_registration_id_tokenization_finished(
                 status=Status.SUCCESS,
                 registration_status=RegistrationStatus(
                     regState="tokenization_finished",
-                    created=datetime.now(),
+                    created=datetime.now().isoformat(),
```

Comment:
I'm assuming this is what the dates look like coming out of the API?

In `littlepay` we have to replace the `Z` ISO offset character in the input datetime string (that we get from the API) with `+00:00` for Python's parsing to handle it correctly: https://github.com/cal-itp/littlepay/blob/main/littlepay/api/funding_sources.py#L47